rust_icu_uloc/src/lib.rs

```diff
@@ -31,6 +32,109 @@ use {
 /// See `ULOC_FULLNAME_CAPACITY`.
 const LOCALE_CAPACITY: usize = 158;
 
+/// [ULocMut] is a mutable companion to [ULoc].
```

Comment:
Would `ULocBuilder` be an accurate name?

Reply:
I don't think so, since the collaboration doesn't fit the build model.